src/main.zig

```diff
@@ -1371,6 +1371,8 @@ fn buildOutputType(
                         linker_opt_bisect_limit = std.math.lossyCast(i32, parseIntSuffix(arg, "-fopt-bisect-limit=".len));
                     } else if (mem.eql(u8, arg, "--eh-frame-hdr")) {
                         link_eh_frame_hdr = true;
+                    } else if (mem.eql(u8, arg, "--no-dynamicbase")) {
```

Comment:
Even though dynamicbase is the defaultt, should we not still handle that case in the CLI? In other words, can this happen?

```
zig build-exe --dynamicbase
```

If so, won't it currently fail with "unknown flag"? Did you check by any chance? Perhaps we should still just match it and ignore since it's the default?

Reply:
Ah, great point. Will update